### PR TITLE
GraphQL API: add `inactiveSince: DateTime` filter parameter to `users` query

### PR DESCRIPTION
**Pull request by mrnugget**
This should help make `src users clean` (added in https://github.com/sourcegraph/src-cli/pull/826) much less resource-intensive and faster by filtering the users that are returned on the server.

`src users clean` will need to be updated to make use of this parameter and only request users that have been inactive since the `-days` flag of the command.

## Usage

```graphql
query InactiveUsers {
  users(inactiveSince: "2022-09-20T10:15:30Z") {
    totalCount
    nodes {
      username
    }
  }
}
```

## Test plan

- Manual testing of this query in API console

### CODE REVIEW

File: cmd/frontend/graphqlbackend/users.go [hunk 1]
```diff
@@ -2,31 +2,40 @@ package graphqlbackend
...
+func (r *schemaResolver) Users(args *usersArgs) *userConnectionResolver {
 	var opt database.UsersListOptions
 	if args.Query != nil {
 		opt.Query = *args.Query
 	}
 	if args.Tag != nil {
 		opt.Tag = *args.Tag
 	}
+	if args.InactiveSince != nil {
+		fmt.Printf("inactiveSince=%s\n", args.InactiveSince.Time)
```
Comment (eseliger) [inline cmd/frontend/graphqlbackend/users.go:36]:
```suggestion
```

**Review by eseliger — APPROVED** [review]

File: cmd/frontend/graphqlbackend/users.go (hunk 1 above)
Comment (mrnugget) [reply cmd/frontend/graphqlbackend/users.go:36]: That was in there for a specific purpose, but _fine_, okay, if you don't want it...

Comment (eseliger) [reply cmd/frontend/graphqlbackend/users.go:36]: I don't, sorry.

Comment (eseliger) [reply cmd/frontend/graphqlbackend/users.go]: use `fmt.Fprintf(os.Stdout`

**Review by DaedalusG — APPROVED** [review]
Looks good! This should much improve src users clean!

**Pull request merged**